### ISSUE TITLE
XWFIM - 404 error on startup

### ISSUE THREAD

**Issue by m-terlinde** [EricZimmerman/Issues#211]
Hello,

currently during startup XWFIM greets me with a 404 error:

![image](https://github.com/EricZimmerman/Issues/assets/10097313/30606e2c-1eb8-4385-b9d2-daa18093ca91)

I've tested three different computers and even a new download of XWFIM produces the 404 error after typing in username and password for the dongle version.

Could you kindly check?

Cheers,
Matthias

**Comment by AndrewRathbun** [comment]
If you run the Get-ZimmermanTools.ps1 script, make sure you run it with -NetVersion 0 added to your command as XWFIM is only a .NET 4 tool and by default Get-ZimmermanTools.ps1 only updates .NET 6 tools. 0 will download both 
.NET 4 and 6 versions of EZ Tools. That will solve your problem!

**Comment by AndrewRathbun** [comment]
TLDR X-Ways changed some paths somewhere and the new version of XWFIM that Eric pushed out recently accommodates those changes.

**Comment by EricZimmerman** [comment]
Or manually download from my site

**Comment by m-terlinde** [comment]
The more you know, thanks!
